dist/src/main/dist/conf/init.sh

```diff
@@ -19,6 +19,9 @@ if [ -d /mnt/ephemeral ] ; then
         sudo chown -R ${user} /mnt/ephemeral/workers/
         ln -s /mnt/ephemeral/workers/ hazelcast-simulator-${version}/workers
      fi
+
```

Comment:
Probably best to guard this with a version check + add some comments what is the problem being solved here. 

I also don't know if the eth0 will always be there on all ec2 instances.